app.js

```diff
@@ -141,69 +277,102 @@ function checkEnemyRocketCollision(enemyX, enemyY) {
 /**
  * Bind all game keyboard controls
  */
+function handleClick (event) {
+	gutling.x = event.pageX;
+	gutling.y = event.pageY;
+  gutling.ammo -= 10;
+  console.log('Gutling Ammo: ' + gutling.ammo);
+  console.log('Rocket Ammo: ' + rocket.ammo);
+}
+
 function handleKeyDown(event) {
   console.log (event.key);
-  if (event.key == "r" || event.key == 'к') {
+  if ((event.key == "r" || event.key == 'к') && rocket.ammo != 0) {
     fireRocket();
     fireBullet();
   }
 
   if (event.key == "a" || event.key == 'ф') {
-    moveShipLeft();
+    isShipMovingLeft = true;
```

Comment:
I think these are @19thanks96 's changes, which are already in `master`.
I don't think you should have them here in your PR for HealthBar.

We can check why are these changes shown here together :) On the lesson or via a video call.

Reply:
I was going to add a keyup event for the airfighter movement animation anyway, but you added it earlier, so I decided to use the existing code for that. What i should do now?